source/MaterialXRenderOsl/OslValidator.h

```diff
@@ -203,17 +206,17 @@ class OslValidator : public ShaderValidator
   protected:
     ///
     /// Shade using OSO input file. Will throw an exception if an error occurs.
-    /// @param outputPath Path to input .oso file.
+    /// @param filePath Path to input .oso file.
```

Comment:
Somewhat misleading docs (I guess search and replace error). I think this should be an output directory, not a path to an input file?

Reply:
Hmm.. or is it an input path folder. This is slightly confusing :)